### DISCUSSION TITLE
Can't see Attacks

### DISCUSSION THREAD

**Discussion by albijoe** [dmdorman/hero6e-foundryvtt#23] [Q&A] [answered]
Can you explain how you are getting attacks into Foundry?  Like, an example of "I made Character A with X,Y, and Z and loaded them into H6e v2, then (did nothing / performed action H) and actions were usable.

  I loaded powers and maneuvers onto a test character, loaded them into a test foundry world, and got nothing.

Notes: 
1. I did not change settings from default for Hero System 6e (Unofficial) V2.
2. I am currently using Hero Designer Build 20210804.

**Comment by aeauseth** [comment, marked as answer]
Currently you upload your HDC file (from Hero Designer) into a token.  It is then up to you to MANUALLY create attacks and power defenses, based on your powers.  Clicking the dice icon for the attack will provide an attack dialog and chat cards.  As of 2.1.1-alpha there is no automation for creating defense/attacks that foundry can use.
